src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+    switch (size) {
+      case 'default':
+        levers.size = 'su-px-26 su-pt-10 su-pb-11 su-text-16 md:su-text-20';
+        break;
+
+      case 'big':
+        levers.size = 'su-px-34 su-py-15 su-text-20 md:su-text-24';
+        break;
+
+      case 'small':
+        levers.size = 'su-px-19 su-py-9 su-text-16 md:su-text-18';
+        break;
+    }
+  }
+
+  // icon
+  let heroicon = '';
+
+  if (icon && ctaLinkIcons.includes(icon)) {
+    switch (icon) {
+      case 'action':
+        heroicon = 'chevron-right';
+        levers.icon = 'su-h-1em su-w-1em su-ml-4 su--mt-2';
+        break;
+
+      case 'more':
+        heroicon = 'arrow-narrow-right';
+        levers.icon = 'su-h-09em su-w-09em su-ml-5 su--mt-2';
+        break;
+
+      case 'external':
+        heroicon = 'arrow-up';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-2 su-transform su-rotate-45 group-hocus:su-rotate-45';
+        break;
+
+      case 'lock':
+        heroicon = 'lock-closed';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-3';
+        break;
+
+      case 'download':
+        heroicon = 'download';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-3';
+        break;
+
+      case 'video':
+        heroicon = 'play';
+        levers.icon = 'su-h-08em su-w-08em su-ml-6 su--mt-3';
+        break;
+
+      case 'email':
+        heroicon = 'mail';
+        levers.icon = 'su-h-08em su-w-08em su-ml-7 su--mt-2';
+        break;
+
+      case 'jump':
+        heroicon = 'chevron-down';
+        levers.icon = 'su-h-1em su-w-1em su-ml-4 su--mt-2';
+        break;
+
+      case 'none':
+        break;
+    }
+  }
+
+  // animate
+
+  // Common classes if animation is needed
+  if ((animate && ctaLinkAnimations.includes(animate)) && animate !== 'none') {
+    levers.animate = 'su-transition-transform group-hocus:su-transform ';
+  }
+
+  // Add specific classes for each type of animation
+  if (animate && ctaLinkAnimations.includes(animate)) {
+    switch(animate) {
+      case 'right':
+        levers.animate += 'group-hocus:su-translate-x-02em';
+        break;
+
+      case 'top-right':
+        levers.animate += 'group-hocus:su-translate-x-01em group-hocus:su--translate-y-01em';
+        break;
+
+      case 'down':
+        levers.animate += 'group-hocus:su-translate-y-02em';
+        break;
+
+      case 'none':
+        break;
+    }
+  }
+
+  return (
+    <a
+      className={clsxd(levers.isButton, 'su-w-fit su-no-underline hover:su-underline focus:su-underline su-group su-transition-colors',
+        levers.display, levers.color, levers.size, levers.variant, classes.link)}
+      href={link}
+      ref={ref}
+      {...props}
+    >
+      {text}
+      {srText &&
+        <span className='su-sr-only'> {srText}</span>
+      }
+      {icon &&
+        <Icon icon={heroicon} type='solid' aria-hidden={true} className={clsxd('su-inline-block', levers.icon, levers.animate, classes.icon)}/>
+      }
```

Comment:
It would be nice if we would allow the developer to pass in their own icon and provide this one as a default.

Reply:
Yeah, this is in the plans 😄